build/CoreFoundation/dll/CoreFoundation.vcxproj

```diff
@@ -152,7 +152,7 @@
       <IgnoreAllDefaultLibraries>false</IgnoreAllDefaultLibraries>
       <GenerateWindowsMetadata>false</GenerateWindowsMetadata>
       <ModuleDefinitionFile>CoreFoundation.def</ModuleDefinitionFile>
-      <AdditionalDependencies>mincore.lib;libxml2.lib;icudt.lib;icuin.lib;icuuc.lib;libdispatch.lib;icudata.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
icudt.dll 7 _kb_ -> 25mb

---
In reply to: [71421042](https://github.com/Microsoft/WinObjC/pull/668#discussion_r71421042) [](ancestors = 71421042,71389811)